scipy/signal/sigtoolsmodule.c

```diff
@@ -934,22 +952,27 @@ PyObject *PyArray_OrderFilterND(PyObject *op1, PyObject *op2, int order) {
 	PyDataMem_FREE(zptr);
 	zptr = PyArray_Zero(ap1);
 	if (zptr == NULL) goto fail;
-	ap1_ptr = ap1->data + offset1*is1;
-	for (k=0; k < ap1->nd; k++) {a_ind[k] = mode_dep[k]; check_ind[k] = ap1->dimensions[k] - ap2->dimensions[k] - mode_dep[k] - 1;}
-	a_ind[ap1->nd-1]--;
+	ap1_ptr = PyArray_DATA(ap1) + offset1*is1;
+	for (k=0; k < PyArray_NDIM(ap1); k++) {
+            a_ind[k] = mode_dep[k];
+            check_ind[k] = PyArray_DIMS(ap1)[k] - PyArray_DIMS(ap2)[k] - mode_dep[k] - 1;
+        }
+	a_ind[PyArray_NDIM(ap1)-1]--;
 	i = PyArray_Size((PyObject *)ret);
 	while (i--) {
-	  /* Zero out the sort_buffer (has effect of zero-padding
-	     on boundaries). Treat object arrays right.*/
+	  /* 
```

Comment:
hmm, both this one and the one above are tabs-vs-spaces. I'll fix things in these two places but am not going to attempt global replace of tabs.